cli/script/command-executor.ts

```diff
@@ -918,6 +923,79 @@ export var release = (command: cli.IReleaseCommand): Promise<void> => {
         });
 }
 
+export var releaseCordova = (command: cli.IReleaseCordovaCommand): Promise<void> => {
+    var platform: string = command.platform.toLowerCase();
+    var projectRoot: string = process.cwd();
+    var platformFolder: string = path.join(projectRoot, "platforms", platform);
+    var platformCordova: string = path.join(platformFolder, "cordova");
+    var outputFolder: string;
+    var preparePromise: Promise<void>;
+
+    if (platform === "ios") {
+        outputFolder = path.join(platformFolder, "www");
+    } else if (platform === "android") {
+        outputFolder = path.join(platformFolder, "assets", "www");
+    } else {
+        throw new Error("Platform must be either \"ios\" or \"android\".");
+    }
+
+    var prepareScript = `
+        var path = require('path');
+        var ConfigParser = require('cordova-common').ConfigParser;
+        var Api = require('./Api');
+        var project = {
+            projectConfig: new ConfigParser(path.join("${projectRoot}", 'config.xml')),
+            root: "${projectRoot}",
+            locations: {
+                www: path.join("${projectRoot}", 'www')
+            }
+        };
+        var preparer = new Api();
+        preparer.prepare(project);
+        `;
+
+    var prepareOptions: { cwd: string } = { cwd: platformCordova };
+
+    try {
+        var prepareProcess: any = spawnSync("node", ["-e", prepareScript], prepareOptions);
+
+        if (prepareProcess.error) {
+            throw prepareProcess.error;
+        }
+    } catch (error) {
+        throw new Error(`Unable to prepare project. Please ensure that this is a Cordova project and that platform "${platform}" was added with cordova platform add "${platform}"`);
```

Comment:
I think the double quotes should be moved:
`was added with "cordova platform add ${platform}"`